src/monte_carlo.c

```diff
@@ -0,0 +1,185 @@
+#include "monte_carlo.h"
+#include "linear_congruential_random_generator.h"
+#include <math.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+
+mc_real calc_UCB(Node* node) {
```

Comment:
I doubt this compiles, Isn't Node a struct? 

Aren't you supposed to write mc_real calc_UCB(struct Node* node) ?

Reply:
Oh, you used typedef, nevermind then.